Common/Models/StatusExtensionContext.swift

```diff
@@ -293,6 +293,7 @@ struct StatusExtensionContext: RawRepresentable {
     var pumpLifecycleProgressContext: DeviceLifecycleProgressContext?
     var cgmStatusHighlightContext: DeviceStatusHighlightContext?
     var cgmLifecycleProgressContext: DeviceLifecycleProgressContext?
+    var carbsOnBoardValue: Double?
```

Comment:
[nit] Any reason for `Value` suffix?

Reply:
my intention here is so that it isn't confused with a value & unit pairing. But I see that didn't help. :) I'll remove.